src/service/viewer-impl.js

```diff
@@ -1005,7 +1005,7 @@ export class Viewer {
       return Promise.reject(getChannelError());
     }
     return this.messagingReadyPromise_.then(() => {
-      return this.sendMessageUnreliable_(eventType, data, awaitResponse);
+      return this.messageDeliverer_(eventType, data, awaitResponse);
```

Comment:
It reduces duplication, so that only `#sendMessageUnreliable` knows how to really send a message. This should continue to call the unreliable method (since we've assured it's now "reliable" by waiting for messaging to be established).